en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md

```diff
@@ -0,0 +1,168 @@
+        
+        <img src="/assets/img/connectors/certificate-and-secrets.png" title="Create certificate and secrets" width="800" alt="Create certificate and secrets"/> 
+    
+    14. Click new **New client secrets**
+    
+        <img src="/assets/img/connectors/new-client-secret.png" title="New client secret" width="800" alt="New client secret"/> 
+     
+    15. Add **Description** and **Expires** values. Click **Add** and copy the value.    
+    
+        <img src="/assets/img/connectors/add-client-secret.png" title="Add client secret" width="800" alt="Add client secret"/>
+    
+    16. Search **Users** inside the Azure Active Directory and Create **New user**.(This user would be linked to the Application User which will create in the Dynamics 365 CRM)    
+        
+        <img src="/assets/img/connectors/add-new-user.png" title="Create new user" width="800" alt="Create new user"/>
+        
+    17. Make sure fill all mandatory fields and click **Create**.    
+       
+        <img src="/assets/img/connectors/create-user.png" title="Fill new user details" width="800" alt="Fill new user details"/>
+
+* ## Setting Up the Application user in Microsoft Dynamics 365 CRM
+    
+    1. Navigate to [Microsoft Dynamics 365 account](https://portal.office.com/) and select **Create a Dynamics 365  Account**.
+    
+    2. Login to created **Dynamics 365 account**. Click **Admin** icon.
+    
+       <img src="/assets/img/connectors/MSdynamics365-admin-icon.png" title="Dynamics365 admin center" width="800" alt="Dynamics365 admin center"/>
+    
+    3. Click **Show all** from the dropdown in the left corner scroll bar.
+    
+       <img src="/assets/img/connectors/MSdynamics365-show-all.png" title="Dynamics365 show all" width="800" alt="Dynamics365 show all"/>
+       
+    4. Click **All admin centers** and click **Dynamics 365** icon. It will navigate to the **Power Platform admin center**
+
+       <img src="/assets/img/connectors/MSdynamics365-all-admin-center.png" title="Dynamics365 all admin center" width="800" alt="Dynamics365 all admin center"/>
+
+    5. Select created environment and click **Open environment**.
+    
+       <img src="/assets/img/connectors/MSdynamics365-select-environment.png" title="Dynamics365 select environment" width="800" alt="Dynamics365 select environment"/>
+       
+    6. Navigate to Settings -> Security -> Users.
+    
+       <img src="/assets/img/connectors/MSdynamics365-settings.png" title="Dynamics365 settings" width="800" alt="Dynamics365 settings"/>
+        
+       <img src="/assets/img/connectors/MSdynamics365-Users.png" title="Dynamics365 users" width="800" alt="Dynamics365 users"/>
+    
+    7. Choose **Application Users** in the view filter.
+    
+       <img src="/assets/img/connectors/MSdynamics365-filter-application-user.png" title="Filter application users" width="800" alt="Filter application users"/>
+    
+    8. Select -> **New**.
```

Comment:
Shall we merge this with the above point?

Reply:
Merged.